### PR TITLE
fix: [CDS-53837]: modified text from single to individual

### PR DESCRIPTION
**Pull request by sahithikolichala**
#### Screenshots

NA

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Sonar: `retrigger sonar`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Feature Name Check: `trigger featurenamecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress Rest: `retrigger cypress-rest`
- Cypress CD: `retrigger cypress-cd`
- Cypress Pipeline: `retrigger cypress-pipeline`
- Cypress CV: `retrigger cypress-cv`
- Fix Prettier: `fix prettier`
</details>

### CODE REVIEW

**Comment by sahithikolichala** [comment]
retrigger coverage

**Comment by sahithikolichala** [comment]
retrigger coverage

**Review by kapilkumawat86 — APPROVED** [review]

File: src/modules/20-rbac/components/UserGroupsInput/UserGroupExpressionInput.tsx [hunk 1]
```diff
@@ -53,11 +53,11 @@ const UserGroupExpressionInput: React.FC<Extended> = props => {
         }}
         selectedValue={formik?.values['userGroupExpression']}
       >
-        <Radio label={getString('common.single')} value={ExpressionTypes.Single} />
+        <Radio label={getString('common.single')} value={ExpressionTypes.Individual} />
```
Comment (rameshx) [inline src/modules/20-rbac/components/UserGroupsInput/UserGroupExpressionInput.tsx:56]: Shouldn't the label also be updated to 'Individual'?

**Comment by sahithikolichala** [comment]
retrigger jest

**Review by kapilkumawat86 — APPROVED** [review]

**Comment by sahithikolichala** [comment]
retrigger jest

**Review by rameshx — APPROVED** [review]

**Pull request merged**